subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/screen/ScreenChecks.kt

```diff
@@ -36,6 +36,7 @@ class SimpleScreenChecks(override val screen: Screen) : ScreenChecks
 
 open class StrictScreenChecks(override val screen: Screen) : ScreenChecks {
 
+    // Delete?
```

Comment:
@eugene-krivobokov Could you explain the neediness of that function?

Reply:
It is for backward compatibility with `ScreenChecks` interface.
Screens that haven't supported strict checks yet, use legacy `SimpleChecks`.

```kotlin
interface Screen {

  val checks: ScreenChecks
        get() = SimpleScreenChecks(this) // TODO: replace by StrictScreenChecks MBS-7204
```

We need this to [support strict checks on screens one by one](https://avito-tech.github.io/avito-android/docs/test_framework/testframework/#screenchecks). After this, we will use only one function.